contracts/Libraries/ERC20.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.8.4;
+
+interface IERC20 {
+    function totalSupply() external view returns (uint256);
+
+    function balanceOf(address account) external view returns (uint256);
+
+    function transfer(address recipient, uint256 amount)
+        external
+        returns (bool);
+
+    function allowance(address owner, address spender)
+        external
+        view
+        returns (uint256);
+
+    function approve(address spender, uint256 amount) external returns (bool);
+
+    function transferFrom(
+        address sender,
+        address recipient,
+        uint256 amount
+    ) external returns (bool);
+
+    event Transfer(address indexed from, address indexed to, uint256 value);
+
+    event Approval(
```

Comment:
@mattdf i think nows a good time to import openzeppelin, a lot of our libs will use interfaces they have declared already. Wdyt @Arachnid

Reply:
I did import it for the actual ERC20 token, I just didn't want to depend on it for the interface to keep it more self contained.

Reply:
I'd say this is a good part to import the interfaces, easier for auditors to know we have the right one etc.

Reply:
kk, i replaced it with an import from zep

Reply:
Agreed, we should use the OZ interfaces wherever practical.